src/photos/ducks/clustering/albums.js

```diff
@@ -100,6 +149,10 @@ const findPhotosToReclusterize = async albums => {
   return photos
 }
 
+const photoExists = (photos, photo) => {
```

Comment:
How about naming this function `photoExistsInCluster` or `photoExistsInCollection` or something like that? Because like… the photo exists, it's right there. I'm not even totally sure we need this function since it's just a wrapper around a one liner?

Reply:
Ok for the renaming. I prefer to keep the function, as the semantic of the name gives a quicker insight than doing `!photos.find(p => p.id === photo.id)`